kerykeion/transits_time_range_factory.py

```
@@ -268,8 +267,9 @@ def get_transit_moments(self) -> TransitsTimeRangeModel:
             transits=transit_moments,
         )
 
-
-    def get_transit_events(self, *, refine_exact_moments: bool = False, refinement_iterations: int = 12) -> "TransitEventsTimeRangeModel":
+    def get_transit_events(
+        self, *, refine_exact_moments: bool = False, refinement_iterations: int = 12
+    ) -> "TransitEventsTimeRangeModel":
```

Ruff (0.15.10)

[error] Undefined name `TransitEventsTimeRangeModel`

(F821)